File: result/repair/starchat/python2java/python_repair/MAXIMUM_SUM_PAIRS_SPECIFIC_DIFFERENCE_1.py
def f_gold(arr, N, k):
        arr.sort()
        maxSum = 0
        for i in range(N - 1):
            if (arr[i]-arr[i-1]<k):
                maxSum += arr[i + 1] + arr[i]
                i -= 1
        return maxSum

File: result/repair/transcoder/python2java/java_repair/MAXIMUM_SUM_PAIRS_SPECIFIC_DIFFERENCE_1.java
  int f_gold ( int [ ] arr , int N , int k ) {
  int maxSum = 0 ;
  ;
  Arrays . sort ( arr ) ;
  int i = N - 1 ;
  while (i>0){
    if ( arr [ i ] - arr [ i - 1 ] < k ) {
      maxSum += arr [ i ] ;
      maxSum += arr [ i - 1 ] ;
      i -- ;
    }
    i -- ;
  }
  return maxSum ;
}

